app/javascript/packages/address-search/components/in-person-locations.spec.tsx

```diff
@@ -90,6 +90,23 @@ describe('InPersonLocations', () => {
     ).to.not.exist();
   });
 
+  it('renders a header at the top of the results', () => {
+    const headingText = 'mock heading';
+    const testId = 'mock-heading';
+    const { getByTestId } = render(
+      <InPersonLocations
+        address={address}
+        locations={locations}
+        onSelect={onSelect}
+        noInPersonLocationsDisplay={NoLocationsViewMock}
+        resultsSectionHeading={() => <h2 data-testid={testId}>{headingText}</h2>}
+      />,
+    );
+
+    expect(getByTestId(testId)).to.exist();
+    expect(getByTestId(testId).textContent).to.equal(headingText);
+  });
+
```

Comment:
Same as above, what do you think about adding a test to ensure this page renders (without additional component) when the resultsSectionHeading prop is not passed in. This is the behavior in IDP so think a test here would be a nice addition.